src/middle/Stan_math_signatures.ml

```diff
@@ -160,17 +176,23 @@ let full_lpmf = [Lpmf; Rng; Ccdf; Cdf]
 let reduce_sum_functions =
   String.Set.of_list ["reduce_sum"; "reduce_sum_static"]
 
-let variadic_ode_functions =
+let variadic_ode_adjoint_fn = "ode_adjoint_tol_ctl"
+
+let nonadjoint_variadic_ode_fns =
```

Comment:
I don't have a strong opinion if it should be `variadic_ode_adjoint` or `adjoint_variadic_ode` but I do feel `adjoint` and `nonadjoint` should be consistent about this.

Reply:
Good call, will go with adjoint_variadic_ode/nonadjoint_variadic_ode

Reply:
Well, I flip flopped and went with variadic_ode_adjoint variadic_ode_nonadjoint just because the function is named ode_adjoint and not adjoint_ode. Fixed.